### ISSUE TITLE
Some small bugs

### ISSUE THREAD

**Issue by Yivan** [dpfaffenbauer/WorkflowGUI#1]
Another really great plugin of you! I tested it and it is really usefull to make easily workflow rules.

I just noticed some small problems. Here they are:

- In "Actions" panel, editing an action : we can only choose one "Users" and one "Notification Users". And they cannot be blancked/erased after. I think making the two field multiselect should handle this.
I think I will open an issue on Pimcore for a new feature: possibility of using "role" too (so creating new user will be automatically targeted to the actions possible for him).

- In "Actions" panel, editing an action : for event, if they are empty (no method filled) they should not be injected in the PHP workflow configuration as it brings an error "Invalid callback provided" when running the action on the object panel.
See this code generated by your plugin in the PHP workflow configuration file:
```
"events" => [
	"before" => [
		"before",
		""
	],
	"success" => [
		"success",
		""
	],
	"failure" => [
		"failure",
		""
	]
]
```
If it is manually removed in the configuration file, it work well, but at every save it come back.
Maybe it is a problem which need to be corrected on the Pimcore core too.

- In "Actions" panel : the delete button is not visible (last stable Pimcore 4.4.2 just released today) - Just tested on Firefox last stable version on W7 desktop.

- Transition definition panel: I think it is not "State" but "Status" the colum label (for Transition to).

As you can see they are just small problems, and everything other work really fine!
Thanks again to share this with us.

**Comment by dpfaffenbauer** [comment]
Should all be fixes without following:

- In "Actions" panel : the delete button is not visible (last stable Pimcore 4.4.2 just released today) - Just tested on Firefox last stable version on W7 desktop.

Could not be reproduced or I didn't understand what you mean there? Do you mean this screen?

<img width="1331" alt="bildschirmfoto 2017-01-22 um 14 19 12" src="https://cloud.githubusercontent.com/assets/5981845/22182769/ca61c024-e0ad-11e6-931f-bd1eb6c6c767.png">

**Comment by Yivan** [comment]
@dpfaffenbauer Thanks a lot for your very fast return on this! : )

For the delete action icon here are two screenshots. It happen only on Firefox (I tested in safemode and same no icon).

Firefox: no icon
![firefox](https://cloud.githubusercontent.com/assets/9025839/22184446/ee345eca-e0d1-11e6-836b-7a069d7b0901.png)

Chrome: there is the icon
![chrome](https://cloud.githubusercontent.com/assets/9025839/22184452/fc0c7f96-e0d1-11e6-821d-a28439d0bc4b.png)

So it is a navigator problem. But starting in safemode don't correct the problem. The firefox/w7 desktop is my usually developpement station and i have no such icon problem on other plugins or Pimcore core.
I think something in the CSS is not compatible. I checked firebug, but i can see the image as background and Firebug show the icon image (so image is here and accessible by the browser...).

Here is the Firefox version:
![version](https://cloud.githubusercontent.com/assets/9025839/22184525/aa4d4dfa-e0d3-11e6-92e6-1ae10956eb40.png)

If it can help I managed to make it appear with commenting the `overflow: hidden` on the parent element:
![patch](https://cloud.githubusercontent.com/assets/9025839/22184522/89d62d12-e0d3-11e6-9246-13d26979f67e.png)
But as it is core ExtJS style, i think it is not the good way...
I try z-index and some other CSS tricks on the pimcore_icon_delete, but it didn't work.

I think maybe the two icons should maybe splited in 2 extjs columns instead of actually just one for both ?

If i can help or make some other test feel free to ask me!
Thanks.
